api/src/main/java/org/openmrs/hl7/handler/ORUR01Handler.java

```diff
@@ -354,13 +354,12 @@ private Message processORU_R01(ORU_R01 oru) throws HL7Exception {
 					
 				}
 				catch (HL7Exception e) {
-					errorInHL7Queue = e;
-				}
+					errorInHL7Queue = e; 
+					}
 				finally {
 					// Handle obs-level exceptions
 					if (errorInHL7Queue != null) {
-						throw new HL7Exception(Context.getMessageSourceService().getMessage(
-						    "ORUR01.error.improperlyFormattedOBX",
+						
 						    new Object[] { PipeParser.encode(obx, new EncodingCharacters('|', "^~\\&")) }, null),
```

Comment:
I think you accidently added a bracket after `null`, try removing that and see if the build is passing, also make sure to squash all commits into 1

Reply:
but sonar say that we should remove it from finally block

Reply:
@PrathameshMutkure

Reply:
Yeah, you have to remove it but also transfer it somewhere else so that we don't change the logic behind it